packages/client/modules/userDashboard/components/OrgBilling/Billing.tsx

```diff
@@ -166,49 +92,56 @@ const ActiveUserBlock = styled('div')({
   paddingTop: 16
 })
 
-const Error = styled('div')<{isError: boolean}>(({isError}) => ({
-  alignItems: 'center',
-  color: isError ? PALETTE.TOMATO_500 : PALETTE.SLATE_600,
-  display: isError ? 'flex' : 'none',
-  lineHeight: '24px'
-}))
-
-const Message = styled('div')({
-  fontSize: 15,
-  paddingLeft: 4
-})
+const stripePromise = loadStripe('pk_test_MNoKbCzQX0lhktuxxI7M14wd')
```

Comment:
This is Stripe's publishable key so it should be OK for it to be publically accessible: https://stripe.com/docs/keys?locale=en-GB#obtain-api-keys

This was the same way they're doing it in their [example](https://stripe.com/docs/payments/quickstart), but I see that we have the key stored on the `window._ACTION__` object so I've updated this to use that!

Reply:
oh yeah, I know it will be visible anyway, I should've been more specific. I just wanted to have a way to use different key on different environments. thanks for the fix!